.storybook/preview.jsx

```diff
@@ -31,8 +33,12 @@ export const parameters = {
   },
 }
 
-import { StorybookComponentWrapper } from '../src/lib/containers/StorybookComponentWrapper'
+// Initialize MSW
+initialize({
+  onUnhandledRequest: 'bypass',
+})
 
+// For now, don't apply the MSW decorator. Only use on a per-story basis.
```

Comment:
If we wanted to use MSW in all stories, we would add `mswDecorator` to the below array.